tests/test_smtp/test_smtp_util.py

```diff
@@ -3,21 +3,39 @@
 import unittest
 from io import BytesIO
 
+import requests
+
 from itk_dev_shared_components.smtp import smtp_util
 from itk_dev_shared_components.smtp.smtp_util import EmailAttachment
 
 
 class TestTreeUtil(unittest.TestCase):
     """Tests relating to the module smtp.smtp_util."""
+    smtp_server = "localhost"
+    smtp_port = 1025
+    http_port = 8025
```

Comment:
Consider getting these from environment variables with fallback to these default values. Then some weirdo not using the defaults will still be able to run the tests.